### PR TITLE
Add Pokedex

### PR DESCRIPTION
**Pull request by solrac97gr**
## Description

this is an extension for search using Pokemon names information of every pokemon using wikidex and poke API
## Type of change

- New extension
## Screenshot
alt screenshot
## Checklist

- [x] I read the [contribution guidelines](contributing.md)

### CODE REVIEW

File: extensions/pokedex/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "name": "pokedex",
+  "title": "Pokedex",
+  "description": "Check the official Pokedex information for every Pokemon",
+  "icon": "pokeball.png",
+  "author": "solrac97gr",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Pokedex",
```
Comment (mattisssa) [inline extensions/pokedex/package.json:11]: Better to use **Search Pokemon** here.

File: extensions/pokedex/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      "subtitle": "Get the Pokemon information",
```
Comment (mattisssa) [inline extensions/pokedex/package.json:12]: And **Pokedex** here

**Review by mattisssa — CHANGES_REQUESTED** [review]
Hey! Interesting extension 😄 

During the review, I found that it just doesn't work at all. Here are a few bullets + check the comments I left about namings.

- There is _bultasaur_ initially but if you type and remove there is nothing shown. We will prevent this and show an empty view, but for now, would be better if you will return an empty array if there are no items found
- Search doesn't work (see the screencast) 
- Errors are not handled

https://user-images.githubusercontent.com/15333662/137401985-f6862bfe-9225-4b36-9c20-5ae297abf969.mp4

**Comment by PitNikola** [comment]
@solrac97gr, I'm gonna close that PR for now. Please feel free to re-open once you have some changes to review 👍

**Pull request closed without merging**